CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog for the Mapbox Maps SDK for Android
 
 Mapbox welcomes participation and contributions from everyone. Please read [`Contributing Guide`](https://github.com/mapbox/mapbox-gl-native/blob/master/CONTRIBUTING.md) to get started.
+
+## 9.4.0-beta.2 - August 18, 2020
+[Changes](https://github.com/mapbox/mapbox-gl-native-android/compare/android-v9.3.0...android-v9.4.0-beta.2) since [Mapbox Maps SDK for Android v9.3.0](https://github.com/mapbox/mapbox-gl-native-android/releases/tag/android-v9.3.0)
+### Dependencies
+ - Update telemetry to 6.1.0
+ - Update core library to 3.0.0
+
 ## 9.4.0-beta.1 - August 6, 2020
```

Comment:
I think we should combine these two change logs in the CHANGELOG.md and have the specific the beta.2 change log in the release tag.

Reply:
So we should have a single 9.4.0-beta.2 change log that combines both beta.1 and beta.2